### PR TITLE
Defender Can Charge From Fortify

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

Defender can now charge from its Fortify stance when both Charge and Fortify are off cooldown.

## Why It's Good For The Game

Makes Fortify more viable and improves its ability to be integrated into Defender combos as a preface/launch point, making Defender play more fluid and flexible.

## Changelog
:cl:
balance: Defender can now charge from its Fortify stance when both Charge and Fortify are off cooldown.
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
Defenders get extra armor by fortifying and one kf the disadvantages of doing so is delay in movement. Why let this fundamental balance be changed? Plus wouldn't that means they negate the standing up time and directly goes to charging wind.

**Comment by hyper2snyper** [comment]
make standing up after charge from fortify take longer.

**Comment by Surrealaser** [comment]
> Defenders get extra armor by fortifying and one of the disadvantages of doing so is delay in standing up. Why let this fundamental balance be changed? Plus wouldn't that means they negate the standing up time and directly goes to charging wind.

This doesn't get around that delay. The cooldown on fortify has to be over before you can charge out of fortify.

File: code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm [hunk 1]
```diff
@@ -135,6 +135,17 @@
 	if(!do_after(X, 0.5 SECONDS, FALSE, X, BUSY_ICON_GENERIC, extra_checks = CALLBACK(src, .proc/can_use_ability, A, FALSE, XACT_USE_BUSY)))
 		return fail_activate()
 
+	var/mob/living/carbon/xenomorph/defender/D = X
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm:138]: image

Please surr, this really helps with readability

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm (hunk 1 above)
Comment (BraveMole) [reply code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm:138]: EXACTLY

**Review by TiviPlus — APPROVED** [review]

**Comment by SplinterGP** [comment]
I think this should have a small tm before merge.

**Pull request merged**